core/workspace_svg.js

```diff
@@ -992,15 +990,13 @@ Blockly.WorkspaceSvg.prototype.addFlyout = function(tagName) {
       }));
   workspaceOptions.toolboxPosition = this.options.toolboxPosition;
   if (this.horizontalLayout) {
-    if (!Blockly.HorizontalFlyout) {
-      throw Error('Missing require for Blockly.HorizontalFlyout');
-    }
-    this.flyout_ = new Blockly.HorizontalFlyout(workspaceOptions);
+    var HorizontalFlyout = Blockly.registry.getClassFromOptions(
```

Comment:
This no longer throws a clear error for the missing require. Are we okay with that?

Reply:
That’s true, so it’s not exactly the right check to check if Blockly.HorizontalFlyout exists any more since it can be some other object that is registered, but we can check if getClassFromOptions returns nothing and throw. 
I would rather deal with this in the registry. It is currently logging a warning. This sounds like something that applies anywhere we request a class from the registry (the whole warn vs throw dilemma). If we want to throw or warn on a case by case basis then we can add another param to the registry call to request a throw if it yields nothing. I would just rather not have to always check if exists and throw.

Reply:
I like having it optionally throw on failure. At the call site we often know whether a failure would be catastrophic--like here--and a centralized throw would be way better.

Reply:
Ok I’ll add that functionality to the registry, and make it throw from this call site

Reply:
Done.